hicommonwealth/commonwealth
# Allow users to trade pinned token in a community

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/10233

## Description of Changes
Allow users to trade the pinned token in a community 

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Go to any Ethereum community you are an admin of and attempt to pin a token (ex: sushi `0x7d49a065d17d6d4a55dc13649901fdbb98b2afba`) via the community integrations setting.
3. Verify the pinned token shows up in the community sidebar as a token widget
4. Verify you are able to swap this token using the uniswap trade modal.

The whole flow should look like this


https://github.com/user-attachments/assets/314034aa-dd10-42df-aebe-ea6a8be5eae8




## Deployment Plan
N/A

## Other Considerations
N/A
